lib/src/messages/message.dart

```diff
@@ -325,15 +327,37 @@ abstract class Message {
       '\r': r'\r',
       '\t': r'\t',
       '\v': r'\v',
-      "'": r"\'",
+      '\'': r"\'",
       r'$': r'\$'
     };
-    return value.splitMapJoin(
+    return escapedBrackets.splitMapJoin(
       '',
       onNonMatch: (String string) => escapes[string] ?? string,
     );
   }
 
+  static String escapeBracketsAndQuotes(String value) {
+    const quote = 39; // Character '
+    const controlCharacters = [39, 123, 125]; // Characters '{}
```

Comment:
This could then read `const controlCharacters = [$leftQuote, $leftBrace, $rightBrace]` and we wouldn't need the trailing eol comment.